"block-closing-brace-space-after" should ignore whitespace after newline

> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

Bug:
There seems to be a subtle difference between the allowing and the forbidding state of the rule:
Allowed > a single space is needed
Disallowed > no white space characters, not even newline are allowed.
I'd have expected a white space to be disallowed and a newline to be ok.

> Which rule, if any, is this issue related to?

`block-closing-brace-space-after`

> What CSS is needed to reproduce this issue?

```scss
.a {}
.b {}

/* also: */
.c {}

.d {}
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "rules": {
    "block-closing-brace-space-after": "never-single-line",
  }
}
```

> Which version of stylelint are you using?

`HEAD`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

gulp-stylelint

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

no

> What did you expect to happen?

No warnings to be flagged.

> What actually happened (e.g. what warnings or errors you are getting)?

The following warnings were flagged:
```shell
test.css
  1:6  error  Unexpected whitespace after "}" of a single-line block   block-closing-brace-space-after
```

<!--- Note: stackoverflow is our preferred QA forum - http://stackoverflow.com/questions/tagged/stylelint -->

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,7 @@
 - Added: `function-max-empty-lines` rule.
 - Added: `declaration-block-no-ignored-properties` rule.
 - Added: `function-url-data-uris` rule.
+- Fixed: `block-closing-brace-newline-after` use of "single space", rather than "newline", in its messages.
 - Fixed: `function-comma-space-after`, `function-comma-space-before`, `function-parentheses-newline-inside` and `function-parentheses-space-inside` now ignore SCSS maps.
 - Fixed: `property-value-blacklist` and `-whitelist` no longer error on properties without a corresponding list entry.
 - Fixed: `font-weight-notation` now ignores `initial` value.

--- a/src/rules/block-closing-brace-newline-after/index.js
+++ b/src/rules/block-closing-brace-newline-after/index.js
@@ -15,9 +15,9 @@ export const ruleName = "block-closing-brace-newline-after"
 
 export const messages = ruleMessages(ruleName, {
   expectedAfter: () => "Expected newline after \"}\"",
-  expectedAfterSingleLine: () => "Expected single space after \"}\" of a single-line block",
+  expectedAfterSingleLine: () => "Expected newline after \"}\" of a single-line block",
   rejectedAfterSingleLine: () => "Unexpected whitespace after \"}\" of a single-line block",
-  expectedAfterMultiLine: () => "Expected single space after \"}\" of a multi-line block",
+  expectedAfterMultiLine: () => "Expected newline after \"}\" of a multi-line block",
   rejectedAfterMultiLine: () => "Unexpected whitespace after \"}\" of a multi-line block",
 })
 

